.gitignore

```diff
@@ -23,6 +23,7 @@ stamp*
 .dirstamp
 *.tar.*
 *.pc
+Makecache
```

Comment:
I suggest using [`git.mk`](https://github.com/behdad/git.mk), but that can happen separately as a future PR.